components/atoms/a-promo-code.vue

```diff
@@ -78,7 +78,7 @@ export default {
   }
   &__button {
     --button-height: 2rem;
-    --button-font-size: 0.6875rem;
+    --button-font-size: var(--font-size--xs);
```

Comment:
this is a variable form SFUI 10.x we are currently running SFUI 7. Please update this and any other place you may have used a variable from the new version.

Reply:
Ok I will update it